### ISSUE TITLE
tools / clean resin vat - exposure not on whole screen

### ISSUE THREAD

**Issue by chri2** [Lite3DP/Lite3DP-Gen-2#14]
I tried the function and here is a picture of what I had:

![2024-10-22-123431](https://github.com/user-attachments/assets/794aeaa2-e9bf-425a-b9e2-ae258e80eb17)

I would have expected that I'd get a square without holes

**Comment by chri2** [comment]
Exposure time has been 45 seconds. Here are pictures of the UV running without vat and of the screen with lights off:

![2024-10-22-200516](https://github.com/user-attachments/assets/801b0b1f-446e-4297-8293-49e0b9d1bf73)

![2024-10-22-200748](https://github.com/user-attachments/assets/73f4a582-30d6-46d8-b72d-d7aeb299572f)

**Comment by Lite3DP** [comment]
@chri2 Yes, please check that there is nothing under the display.
Also remember that both the mask display and the touchscreen show the same thing, just that due to the lack of backlight nothing is visible on the mask display. Carefully, with the machine disassembled (without cover or base), you can use a flashlight to illuminate the mask display while you navigate the menu and see better where those spots are coming from. In a very rare case, they may be dead areas on the display, hopefully this is not the case.

**Comment by chri2** [comment]
O.k. I took the printer apart and took some pictures. I used some led lights to put lights to the back of the screen while taking the pictures.

## screen off

In these pictures the screen is off and the printer is not powered.

![2024-10-23-113330](https://github.com/user-attachments/assets/5875de75-9e8f-4cf9-b55f-9dfa6b25a0c8)
![2024-10-23-113210](https://github.com/user-attachments/assets/c159cf6f-324f-4fb7-9fa7-7dd3cec093d3)
![2024-10-23-113124](https://github.com/user-attachments/assets/e0b692e3-be83-4410-a399-ba1835cb63b8)
![2024-10-23-113056](https://github.com/user-attachments/assets/e2856422-f4fa-49d2-a27a-d2a6942c721e)
![2024-10-23-112527_6](https://github.com/user-attachments/assets/3bae3664-d3a0-47f6-a2e0-11be68ea83e3)
![2024-10-23-112527_3](https://github.com/user-attachments/assets/c123bcfe-c414-4633-94ba-75cb7bf67913)
![2024-10-23-112527_2](https://github.com/user-attachments/assets/1f9490bb-4114-4a49-8f51-724971adbc39)

## screen active

I powered the printer and took these pictures.

![2024-10-23-113528](https://github.com/user-attachments/assets/f247719a-9d13-4cd0-a9b8-43ffea88d149)

The following pictures are taken while the clean vat tool has been active.


![2024-10-23-113627](https://github.com/user-attachments/assets/17da515e-5504-4712-a090-73d0fdae619c)
![2024-10-23-113621](https://github.com/user-attachments/assets/405cc940-54f1-4944-9329-ff2e404ca311)
![2024-10-23-113609](https://github.com/user-attachments/assets/7e50f592-2b68-437e-87d8-cad556b177d1)

## screen inspection from the back

I carefully controlled the back of the screen for dirt or any signs that may explain the problem seen in the pictures above, but didn't find any. Looking at the silvery back of the screen in daylight the errors from the pictures above can barely be seen as shadows - nearly not notable. I think the visual effect is so weak I can't put it into a photo.

![2024-10-23-114513](https://github.com/user-attachments/assets/274ac1a1-7f8f-49a0-b89c-d04520163ddb)
![2024-10-23-114558](https://github.com/user-attachments/assets/cf16373d-3f63-4f2e-b0fb-d3a993faf2cd)

Please let me know if I can provide more information to help solve the problem.

**Comment by Lite3DP** [comment]
Sorry @chri2 they look like two dead zones on the LCD. Could it be that the edge of the platform hit the display during leveling? I haven't seen anything like this before. It's not the best, but you can still use the machine preventing print sections from including dead zones. However, you might want to order a replacement LCD display from CS before they run out. Sorry again.

**Comment by chri2** [comment]
> Sorry @chri2 they look like two dead zones on the LCD. Could it be that the edge of the platform hit the display during leveling? 

No. And it doesn't look like cracks neither like hits.

> I haven't seen anything like this before. It's not the best, but you can still use the machine preventing print sections from including dead zones. However, you might want to order a replacement LCD display from CS before they run out. Sorry again.

Uhm, I'd say this is a production error. I do not see any way how one could inflict a damage like that - like you probably do neither. I'd expect that I'd get provided a screen as a replacement given that I'm able to change it (which probably is possible since there are screens to order).

I do not see where my fault should be in this that I'd have to pay for the replacement screen?

**Comment by Lite3DP** [comment]
Customer support contact page: https://www.crowdsupply.com/contact

**Comment by chri2** [comment]
Thanks for the suggestion! I did that and am waiting for their answer.

**Comment by chri2** [comment]
@Lite3DP, could you provide some advice/guidance on how to remove the screen from the mainboard of the Lite3DP Gen2? Is it glued to the mainboard? Does heat help to free it?

**Comment by chri2** [comment]
When I exchanged the screen I found that it is easily peeled off the pcb. It seems only to be held in place by adhesion.

Before removing the screen I opened the [THD THD0515-45CL-SN flat cable connector](https://www.lcsc.com/product-detail/FFC-FPC-Flat-Flexible-Connectors_THD-THD0515-45CL-SN_C283158.html) by carefully pulling up the flap of the [Front Flip-Lock Actuator](https://www.te.com/en/products/connectors/pcb-connectors/wire-to-board-connectors/ffc-fpc-ribbon-connectors/fpc-connectors/intro-to-fine-pitch-fpc-connectors.html) with my fingernail.

The new screen I carefully moved in its place maneuvering its flat cable into the **zero insertion force** (ZIF) connector that I opened to remove the old screen.

My new screen had on its back (silver side) *two* foils: 
* one is large and has a corner to grab to peel it off which I did
* the other one (I guess it is supposed to stick on the bigger first one and get peeled off when pulling the thicker one off, but that didn't work for me) I very carefully lifted with a plastic card at one corner until I could grab it with my fingers.

The upper side (black) had a foil installed also.

My new screen works and has no broken areas.
